README.md

```diff
@@ -1,6 +1,6 @@
 JS Exercise
 ===================================
-![Assignment Checks](https://github.com/IT3049C/2.JS-Exercise/workflows/Assignment%20Checks/badge.svg)
+![Assignment Checks](https://s///github.com/IT3049C-Fall20/2-javascript-exercise-tranjg/workflows/Assignment%20Checks/badge.svg)
```

Comment:
This should be https://github.com/IT3049C-Fall20/2-javascript-exercise-tranjg/workflows/Assignment%20Checks/badge.svg